src/microbe_stage/editor/PatchMapNode.cs

```diff
@@ -83,6 +95,12 @@ public Patch Patch
 
     public Action<PatchMapNode>? SelectCallback { get; set; }
 
+    /// <summary>
+    ///   True if the patch is adjasent to a explored patch
+    ///   but has not been entered by the player
+    /// </summary>
+    public bool IsUnknown { get; set; }
```

Comment:
Does it make sense to have something be Discovered and IsUnnknown at the same time? If not I think one variable that's an enum should be used instead to dictate the overall state of a patch.

Also regarding the state I just remembered something. The player can always move back to the patch they started at in the editor, so I think the patch map needs to store the previous exploration states to allow resetting the map if the player decides to go back. Without that the player can explore multiple patches per editor cycle by moving to one patch and then going back to the patch they were in previously and then repeating that action.